packages/driver/src/cy/net-stubbing/events/request-received.ts

```diff
@@ -0,0 +1,199 @@
+import _ from 'lodash'
+
+import {
+  Route,
+  Request,
+  CyHttpMessages,
+  StaticResponse,
+  SERIALIZABLE_REQ_PROPS,
+  NetEventFrames,
+} from '../types'
+import {
+  validateStaticResponse,
+  getBackendStaticResponse,
+  parseStaticResponseShorthand,
+} from '../static-response-utils'
+import $errUtils from '@packages/driver/src/cypress/error_utils'
```

Comment:
:laughing: it was originally in a different package, but i couldn't get the driver types to work in a non-driver package, so i moved it. guess i missed this, yes it can be updated